src/hooks/useFirestoreState.ts

```diff
@@ -0,0 +1,9 @@
+import { DocumentHook, useDocument } from "react-firebase-hooks/firestore";
+import { DocumentData, doc } from "firebase/firestore";
```

Comment:
can you preface type for `import { type DocumentHook, useDocument }` as well as for `DocumentData`